frontend/app/src/entities/events/ui/global-events.tsx

```diff
@@ -22,11 +25,39 @@ export const GlobalEvents = () => {
     return edge.node;
   });
 
+  const count = data?.data?.[INFRAHUB_EVENT]?.count;
```

Comment:
this kind of destructuring are better to be done in `useEvents` directly. The goal is to have data read to use instantly when calling `useEvents`